### PR TITLE
[MAINT] Add 3.10 unit test, compatibilities in setup.cfg

### LINKED ISSUE #817: Add Python 3.10 Support
### Summary
With the newest release of Python, we should add 3.10 support.

### Next Steps
- [ ] Add 3.10 to allowed python versions
- [ ] Include 3.10 unit tests

### PR DESCRIPTION
**Pull request by jbteves**
Closes #817  .

Changes proposed in this pull request:

- Adds 3.10 Support, testing

### CODE REVIEW

**Comment by tsalo** [comment]
It looks like [v1.21.4](https://github.com/numpy/numpy/releases/tag/v1.21.4) supports 3.10.

**Comment by handwerkerd** [comment]
It looks like SciPy 1.8 may now support Python 3.10. Worth confirming? https://docs.scipy.org/doc/scipy/release.1.8.0.html

**Comment by jbteves** [comment]
No dice.

**Comment by tsalo** [comment]
I think this might be worth another check. Can you update from main?

**Comment by jbteves** [comment]
Finally passes, now that we have 3.11!

File: setup.cfg [hunk 1]
```diff
@@ -18,6 +18,8 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
```
Comment (tsalo) [inline setup.cfg:22]: Since 3.11 tests weren't passing, you should probably remove this.

Comment (jbteves) [reply setup.cfg]: Done in 11f3224

**Review by tsalo — APPROVED** [review]
LGTM! Thanks!

**Review by eurunuela — APPROVED** [review]
LGTM, thanks!

**Pull request merged**